code/style.css

```diff
@@ -1,7 +1,490 @@
+
+* {
+
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+}
+
+body {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+} 
+
+section {
+
+  flex: 1;
+  width: 100%
+}
+
+/* HEADER */
+
+nav {
+
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  z-index: 1;
+  position: fixed; 
+
+  background-color: black;
+  font-family: 'Kumbh Sans', sans-serif;
+  
+}
+
+nav a {
+  padding: 1rem;
+  color:lightsalmon;
+  align-self: center;
+}
+
+/* 
+nav > * {
+
+} */ */
+
+
+#header-section {
+
+  /* align-self: center; */
+  width: 100%;
+  height: 50%;
+  display: flex;
+
+  text-align: center;  
+  align-items: center;
+
+  position: relative;
+  
+  justify-content: center;
+  overflow: hidden;
+
+
+}
+
+.heading-text {
+  z-index: 2;
+  
+}
+
+video {
+  object-fit: cover;
+  background-size: contain;
+  background-repeat: no-repeat;
+  z-index: 0;
+  width: 100vw;
+}
+
+
+
+/* FAQ */
+
+article {
+  margin: 0.3rem;
+}
+
+#accordion {
+  flex: 1;
+  min-width: 100%;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(102, 12, 62);
+  padding: 5rem;
+
+}
+
+.questions-box {
+
+  display: flex;
+  flex-direction: column;
+  max-width: 70%;
+  align-self: center;
+
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
 .active + .answer {
-  /* some styling here */
+    /* some styling here */
+    display: block;
+    /* background-color: seashell; */
+
+}
+
+.question {
+
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 600;
+  font-size: 1.4rem;
+  color: #c23bce;
+}
+
+
+.answer {
+  align-self: center;
+  padding: 2rem 5rem 2rem 5rem;
+  color: #fd99d7;
+  display: none;
+  font-family: 'Kumbh Sans', sans-serif;
+  text-align: center;
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+button {
+  background: none;
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
```

Comment:
I can see that most of the css is written with a space between your attribute/value and its declarations but there are some places where it looks different, maybe you could take a quick check that everything looks cohesive before you are done